test/simple/test-child-process-stdout-flush-exit.js

```diff
@@ -52,14 +52,17 @@ if (process.argv[2] === 'child') {
 
   // check if we receive both 'hello' at start and 'goodbye' at end
   child.stdout.setEncoding('utf8');
+  var messageSequence = 0;
   child.stdout.on('data', function (data) {
-    if (data.slice(0, 6) == 'hello\n') {
+    if ((0 == messageSequence ) && (data.slice(0, 6) == 'hello\n')) {
```

Comment:
@mdawsonibm Right, my comment on the rvalue on the left of the equality operator was just to provide some additional context around common practices. I'm fine with keeping it, especially in a test file.